Sources/BranchSDK/BNCServerInterface.m

```diff
@@ -125,9 +125,9 @@ - (void)genericHTTPRequest:(NSURLRequest *)request retryNumber:(NSInteger)retryN
             } else {
                 if (status != 200) {
                     if ([NSError branchDNSBlockingError:underlyingError]) {
-                        [[BranchLogger shared] logWarning:[NSString stringWithFormat:@"Possible DNS Ad Blocker. Giving up on request with HTTP status code %ld", (long)status] error:underlyingError];
+                        [[BranchLogger shared] logError:[NSString stringWithFormat:@"Possible DNS Ad Blocker. Giving up on request with HTTP status code %ld", (long)status] error:underlyingError];
                     } else if ([NSError branchVPNBlockingError:underlyingError]) {
-                        [[BranchLogger shared] logWarning:[NSString stringWithFormat:@"Possible VPN Ad Blocker. Giving up on request with HTTP status code %ld", (long)status] error:underlyingError];
+                        [[BranchLogger shared] logError:[NSString stringWithFormat:@"Possible VPN Ad Blocker. Giving up on request with HTTP status code %ld", (long)status] error:underlyingError];
```

Comment:
Can this message get bubbled up to the error callback?

Reply:
I think it may be fine if not, we'll just have to inform anyone wanting to check for errors to not over count.

Reply:
Yeah, if we created a new, custom error that contained this as the error message